lib/config/validation.ts

```diff
@@ -269,6 +269,14 @@ export async function validateConfig(
             if (key === 'extends') {
               for (const subval of val) {
                 if (is.string(subval)) {
+                  try {
+                    await getPreset(subval, config);
+                  } catch (e) {
```

Comment:
we need to differenciate the error. it should be at least logged as warning

Reply:
you can see examples in the issue https://github.com/renovatebot/renovate/issues/15999

Reply:
@viceice what do you mean Differentiate the error? if we catch the error it will show as Preset not found error,
it gets logged anyway because im pushing it into `errors`

Reply:
yes, log it. as it can be a 404 or invalid json or any other error. so a user should be able to dig and fix the cause